packages/amplify-cli/src/execution-manager.ts

```diff
@@ -354,7 +353,9 @@ export const raiseEvent = async (context: Context, args: AmplifyEventArgs): Prom
         };
         return eventHandler;
       });
-    await sequential(eventHandlers);
+    for (const eventHanler of eventHandlers) {
```

Comment:
This change was required to fix some strange behavior in the prePushHook event. Without this change, `ctrl+c` during the prompt for a missing env var (existing prompt) would not exit the CLI, instead exiting the prompt and the CLI continued executing.

Reply:
Spelling: hanler->handler